chronosphere/registry/registry.go

```diff
@@ -275,11 +284,12 @@ var Resources = mustValidate([]Resource{
 		DryRun: true,
 	},
 	{
-		Name:              "resource_pools_config",
-		Entity:            "ResourcePoolsConfig",
-		API:               V1,
-		Schema:            tfschema.ResourcePoolsConfig,
-		NonStandardEntity: true,
+		Name:        "resource_pools_config",
+		Entity:      "ResourcePools",
+		API:         V1,
+		Schema:      tfschema.ResourcePoolsConfig,
+		SingletonID: "ResourcePoolsConfigID",
```

Comment:
Didn't it use to be:
```
const ResourcePoolsConfigID = "resource_pool_singleton"
```
Is there a backwards compatibility reason to keep these the same?

Reply:
The SingletonID was referencing the constant variable by name rather than the value, but that's a little confusing, so decided to inline the value here, and drop the const (if it's not used anywhere else).